commands/show.go

```diff
@@ -1,22 +1,25 @@
 package commands
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
-	"strings"
-
+	"github.com/MichaelMure/git-bug/bug"
```

Comment:
sorting those imports would be nice

Reply:
Goland does that for you if you ask nicely:
![image](https://user-images.githubusercontent.com/294669/85392690-31734000-b54c-11ea-91aa-4535ee7494e9.png)